fabric-fluid-v1/src/main/java/net/fabricmc/fabric/Action.java

```diff
@@ -0,0 +1,24 @@
+package net.fabricmc.fabric;
+
+/**
+ * describes the behavior of a fluid transaction
+ */
+public enum Action {
```

Comment:
TransactionAction

Reply:
too long :irritatered:

Reply:
action bad and worst

Reply:
give more balanced name

Reply:
`MovementType`, `TransactionDescriptor`, `ActionType`, `FluidMovementType`, `MovementKind`...

`TransactionType`?
`MovementType`?

`TransAction`?

Reply:
all 2 word names, do not give in to the 🖥️✋

Reply:
imo Action is just fine, it doesn't need to be super specific

Reply:
`Action` is vague. Two words describe it properly.

`SimulationType`?